### ISSUE TITLE
[overrides] now support overrides on class members relatived rules?

### ISSUE THREAD

**Issue by xiyuanyuan** [typescript-eslint/typescript-eslint#3594]
- [x] I have tried restarting my IDE and the issue persists.
- [x] I have updated to the latest version of the packages.
- [x] I have [read the FAQ](https://github.com/typescript-eslint/typescript-eslint/blob/master/docs/getting-started/linting/FAQ.md) and my problem is not listed.

**Repro**

```JSON
{
  "rules": {
    "@typescript-eslint/<rule>": ["<setting>"]
  }
}
```

```TS
class Animal {
  getName() { return ''; }
}
export default class Dog extends Animal {

  override bak() {
    return 'wang';
  }

  override getName() { // ok
    console.log('11111 dog')
    return 'wang';
  }
}

```
I can not find related rules to lint my code,only have ts checker info
> This member cannot have an 'override' modifier because it is not declared in the base class 'Animal'.ts(4113)

**Expected Result**

**Actual Result**

**Additional Info**

**Versions**

| package                            | version |
| ---------------------------------- | ------- |
| `@typescript-eslint/eslint-plugin` | `4.27.0` |
| `@typescript-eslint/parser`        | `4.27.0` |
| `TypeScript`                       | `4.3.2` |
| `prettier`                           | `2.3.0` |
| `node`                             | `X.Y.Z` |

**Comment by bradzacher** [comment]
What are you asking for?
Please be lear an explicit. Your request currently makes absolutely no sense.
